Dockerfile.template

```diff
@@ -59,7 +59,7 @@ ENV HAPROXY_SHA256 {{ .sha256 }}
 		# Since 5.4 is supported on haproxy, better use it now, but only for
 		# newer versions since there could be some minor incompatibilities
 		# for existing scripts: https://www.lua.org/manual/5.4/manual.html#8
-		if ([ "2.0", "2.2", "2.4", "2.6", "2.7", "2.8" ] | index(env.version)) then
```

Comment:
2.0 is also out of LTS and also EOL

Reply:
2.0 is not EOL.

Reply:
i found 2 sources that said it is and one that was ambiguous. It can be done in a separate PR

Reply:
I don't know what sources you used, but the official project homepage does not list HAProxy 2.0 as EOL: https://www.haproxy.org/. And neither does yesterday's release announcement of HAProxy 2.0.35 (whereas the one for 2.7.12 did): https://www.mail-archive.com/haproxy@formilux.org/